docs/gno-infrastructure/setting-up-a-local-chain.md

```diff
@@ -10,8 +10,59 @@ In this tutorial, you will learn how to start a local Gno node (and chain!).
 Additionally, you will see the different options you can use to make your Gno instance unique.
 
 ## Prerequisites
+- **Git**
+- **`make` (for running Makefiles)**
+- **Go 1.21+**
+- **Go Environment Setup**:
+  - Make sure `$GOPATH` is well-defined, and `$GOPATH/bin` is added to your `$PATH` variable.
```

Comment:
This should be pretty standard stuff, not sure why we'd need to mention it

Reply:
Maybe something like "Ensure you have Go set up as outlined in the [Go official installation documentation](https://go.dev/doc/install) for your environment" ?

Reply:
This is pretty standard stuff but this is where most people get stuck in the beginning, as experience showed. That is why I suggest keeping it in.

Reply:
I mean, the tutorial is about building from source, not using a ready-made binary, so a valid Go setup is assumed -- if you want to keep it, we should probably have it for every single tutorial that involves building go binaries (all of them, actually, since we don't have a tutorial that uses a release binary).

I don't see a reason for keeping it (or at least having it this detailed, I like @gfanton's suggestion)

Reply:
Okay, I switched it out for @gfanton's proposal: [abd1169](https://github.com/gnolang/gno/pull/2212/commits/abd11698306896f60c9e851ac0213b86d802dac2)